react/features/subtitles/actions.web.js

```diff
@@ -0,0 +1,18 @@
+import { toggleDialog } from '../base/dialog/actions';
+
+import LanguageSelectorDialogWeb from './components/LanguageSelectorDialog.web';
+
+export * from './actions.any';
+
+/**
+ * Signals that the local user has toggled the LanguageSelector button.
+ *
+ * @returns {{
+ *      type: UPDATE_TRANSLATION_LANGUAGE
+ * }}
+ */
+export function toggleLangugeSelectorDialog() {
```

Comment:
A typo here :D

Reply:
I just moved the code around and can't be bothered right now, sorry!